sdv/multi_table/utils.py

```diff
@@ -7,10 +7,18 @@
 import pandas as pd
 
 from sdv._utils import _get_root_tables
+from sdv.errors import InvalidDataError, SamplingError
 from sdv.multi_table import HMASynthesizer
 from sdv.multi_table.hma import MAX_NUMBER_OF_COLUMNS
 
 MODELABLE_SDTYPE = ['categorical', 'numerical', 'datetime', 'boolean']
+RANDOM_STATE = 42
+
+
+def _get_child_tables(relationships):
```

Comment:
Since we are modifying this function, can we take this chance to centralize this function? It seems there are similar functions in 3 places:
- https://github.com/sdv-dev/SDV/blob/1be52f6af74670fef2fe93ef047875def574036a/sdv/multi_table/base.py#L117-L122
- https://github.com/sdv-dev/SDV/blob/447d0d0f5b85b3fae3a0fc7a737915817e8742e0/sdv/_utils.py#L363-L366

Reply:
Yes, done in [43d4441](https://github.com/sdv-dev/SDV/pull/1928/commits/43d4441ccbd89795a56df13d9b1ff99b2465292f)